test/regressions/site/src/tests/Icon/AccentIcon.js

```diff
@@ -0,0 +1,10 @@
+// @flow weak
+
+import React from 'react';
+import Icon from 'material-ui/Icon';
+
+export default function AccentIcon() {
```

Comment:
I have noticed an issue with those really *small unitaire** visual regression tests.
We pay an approximate fix 500ms build time increase for each file.
Meaning that PR is increasing it by +3 second. Not much, but tests after the others, that's gonna start making a different. It's directly linked to our approach.
On the other hand, we have the *monolithic* approach I have tried with the `Tabs` component.
It takes 700ms build time.

What do you think about changing a bit our pattern and aggregating similar variation on a single screenshot?